### PR TITLE
fixing dvtm to be available for elcap+

### PR DESCRIPTION
**Pull request by spacelatte**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --new-formula <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

File: Formula/dvtm.rb [hunk 1]
```diff
@@ -11,8 +11,10 @@ class Dvtm < Formula
     sha256 "88bffcbc907f9ffa900331acf5f994af07f5c1787c328ac56c72935813f57b92" => :mavericks
   end
 
-  # https://github.com/martanne/dvtm/issues/19
-  depends_on MaximumMacOSRequirement => :yosemite
+  patch do
+    url "https://raw.githubusercontent.com/Homebrew/formula-patches/master/dvtm/patch-vt.h-compat.patch"
```
Comment (MikeMcQuaid) [inline Formula/dvtm.rb]: Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!

**Comment by pvtmert** [comment]
okay done, https://github.com/martanne/dvtm/issues/30

**Comment by DomT4** [comment]
@BrewTestBot test this please

**Comment by MikeMcQuaid** [comment]
@pvtmert "The following should work: CFLAGS=-D_DARWIN_C_SOURCE make", could you try something similar instead using `ENV.append_to_cflags "-D_DARWIN_C_SOURCE"`?

**Comment by DomT4** [comment]
I remembered something, we didn't block this with an MMOSR because it didn't build, we blocked it because it had runtime problems under (the presumption was) SIP. The previously linked issue in the formula notes this: https://github.com/martanne/dvtm/issues/19. Has that changed here?

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by MikeMcQuaid** [comment]
@pvtmert Would like to see more conversation on https://github.com/martanne/dvtm/issues/30 before merging this.

**Pull request closed without merging**